app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -125,9 +132,71 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
         clearEstimates()
     }
 
+    override fun onPause() {
+        super.onPause()
+        savedInstanceState = Bundle()
+            //included this boolean value to avoid insufficient balance error if true onResume
+            savedInstanceState?.putBoolean(Constants.SEND_MAX, sendMax)
+
+            //fetch and save the selected source account
+            val selectedSourceAccount = amountHelper.selectedAccount
+            savedInstanceState?.putSerializable(Constants.SELECTED_SOURCE_ACCOUNT, selectedSourceAccount)
+
+            //fetch and save the selected destination account
+            val selectedDestAccount = destinationAddressCard.destinationAccountSpinner.selectedAccount
+            savedInstanceState?.putSerializable(Constants.SELECTED_DESTINATION_ACCOUNT, selectedDestAccount)
+
+            //save destination address state depending on if spinner or address input was selected
+            if(destinationAddressCard.isSendToAccount){
+                savedInstanceState?.putBoolean(Constants.SPINNER, true)
+            } else {
+                savedInstanceState?.putBoolean(Constants.SPINNER, false)
+            }
+    }
+
     override fun onResume() {
         super.onResume()
         destinationAddressCard.addressInputHelper.onResume()
+        if (savedInstanceState != null) {
+            //fetch saved sendMax Value
+            val max = savedInstanceState!!.getBoolean(Constants.SEND_MAX)
+            sendMax = max
```

Comment:
```suggestion
            sendMax = savedInstanceState!!.getBoolean(Constants.SEND_MAX)
```
Remove the redundant variable.